stbt_rig.py

```diff
@@ -967,16 +970,13 @@ def _iter_portal_auth_tokens(self):
                 yield out
         except ImportError:
             sys.stderr.write(
-                "Install the python keyring package so you don't need to "
+                "Install the python \"keyring\" package so you don't need to "
                 "enter your API token every time\n")
 
         while True:
             token = ask('Enter Access Token for portal %s: ' % self.portal_url)
             if not token:
-                # EOF
-                sys.stderr.write("EOF!\n")
-                sys.stderr.flush()
-                break
+                continue
```

Comment:
Note: If I press Control-D, both Python 3's `input` and Python 2's `raw_input` raise [EOFError](https://docs.python.org/3.10/library/exceptions.html?highlight=eoferror#EOFError).